### PR TITLE
Fix host_os literal for windows

### PR DESCRIPTION
**Pull request by aam**

### CODE REVIEW

File: DEPS [hunk 1]
```diff
@@ -444,10 +444,10 @@ hooks = [
   {
     'name': 'frontend_server_packages',
     'pattern': '.',
-    'condition': 'host_os == "windows"',
+    'condition': 'host_os == "win"',
     'cwd': 'src/flutter/frontend_server/',
     'action': [
-      '../../../src/third_party/dart/tools/sdks/win/dart-sdk/bin/pub.bat', 'get',
```
Comment (zanderso) [inline DEPS]: I thought forward slashes would be okay. Is that not the case @bkonyi

Comment (aam) [reply DEPS]: This is how failure with forward slashes looks like:
```
Running hooks:  80% ( 8/10) frontend_server_packages
________ running '../../../src/third_party/dart/tools/sdks/win/dart-sdk/bin/pub.bat get' in 'C:\src\flutter\engine\src/flutter/frontend_server/'
'..' is not recognized as an internal or external command,
operable program or batch file.
Error: Command '../../../src/third_party/dart/tools/sdks/win/dart-sdk/bin/pub.bat get' returned non-zero exit status 1 in C:\src\flutter\engine\src/flutter/frontend_server/
```

Comment (bkonyi) [reply DEPS]: I'm pretty sure they are for DEPS files. I don't recall having any issues with them using forward slashes on Windows.

**Comment by Hixie** [comment]
Sorry about that!

**Pull request merged**